dali/pipeline/operators/crop/bbox_crop.h

```diff
@@ -50,21 +50,35 @@ class RandomBBoxCrop : public Operator<Backend> {
   using Crop = BoundingBox;
   using BoundingBoxes = std::vector<BoundingBox>;
 
+  struct ProspectiveCrop {
+    bool success_;
```

Comment:
Coding guideline, thank Google, doesn't require trailing underscores on _struct_ members (only class). I thing we should cherish that as much as we can while big G is not watching.

Reply:
Done